src/appleseed/foundation/meta/tests/test_aabb.cpp

```diff
@@ -537,3 +526,85 @@ TEST_SUITE(Foundation_Math_AABB)
         EXPECT_TRUE(bbox1 != bbox3);
     }
 }
+
+TEST_SUITE(Foundation_Math_IntegralAABB)
+{
+    TEST_CASE(TestInvalidateOnNoIntersectionOnIntegralAABB)
```

Comment:
The name of this test is incorrect: it makes it sounds like you test `invalidate()` but you don't, you actually test `intersect()`. Review and fix other unit tests.